healvis/beam_model.py

```diff
@@ -33,6 +39,51 @@ def airy_disk(za_array, freqs, diameter=15.0, **kwargs):
     return beam
 
 
+def smooth_beam(freqs, beam_array, freq_ls=2.0, noise=1e-10, output_freqs=None):
+    """
+    Smooth a beam across frequency using a Gaussian Process
+
+    Args:
+        freqs : 1D ndarray
+            Frequency array [Hz]
+        beam_array : 2D ndarray
+            Beam map with shape (Nfreqs, Npixels)
+        freq_ls : float
+            Frequency lengthscale to smooth at [MHz]
```

Comment:
Is there any particular reason to specify this in MHz instead of Hz?

Reply:
Mostly because any reasonable smoothing scale would be on the ~ MHz level, so someone entering it by hand might appreciate the shorthand, but if you think its better to be consistent with [Hz] units of other frequency arguments i could be persuaded